burn-core/src/record/tensor.rs

```diff
@@ -112,12 +112,12 @@ impl<B: Backend, const D: usize> Record for Tensor<B, D, Int> {
         IntTensorSerde::new(self.into_data().convert().serialize())
     }
 
-    fn from_item<S: PrecisionSettings>(item: Self::Item<S>) -> Self {
+    fn from_item<S: PrecisionSettings>(item: Self::Item<S>, _device: &B::Device) -> Self {
         Tensor::from_data(item.data.convert(), &B::Device::default())
```

Comment:
Shouldn't device be passed instead of `&B::Device::default()`?

Reply:
Yes, thanks for spotting that mistake 🙏